fluent-bit.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: fluent-bit
+  version: 2.0.8
+  epoch: 0
+  description: Fast and Lightweight Log processor and forwarder
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: Apache-2.0
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - flex
+      - glibc
+      - openssl-dev
+      - yaml-dev
+      - zlib-dev
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/fluent/fluent-bit/archive/v${{package.version}}.tar.gz
+      expected-sha256: 8ff5566389033669feabc9c69a5c6f417dad5c8b066454388e6a706507262acf
+  - runs: |
+      cmake -B build \
+            -DCMAKE_INSTALL_PREFIX=/usr \
+            -DCMAKE_INSTALL_LIBDIR=lib \
+            -DFLB_TESTS_INTERNAL=On \
```

Comment:
You're not running the tests below so it will speed up the build to disable this. If you want the tests then I think we need to invoke the `make tests` target but you could also run them separately.

Reply:
There are also runtime tests but that may need more setup to execute: `-DFLB_TESTS_RUNTIME=On`

Reply:
I think i will disable tests. Lets leave those upstream